android/quest/src/test/assets/configs/app/application_config.json

```diff
@@ -11,12 +11,7 @@
   ],
   "useDarkTheme": false,
   "syncInterval": 30,
-  "syncStrategy": [
-    "Organization",
-    "Location",
-    "CareTeam",
-    "Practitioner"
-  ],
+  "syncStrategies": ["Organization", "Location", "CareTeam", "Practitioner"],
```

Comment:
Can we have a test to show what happens if we just have 1 syncStrategy defined?

Reply:
Definitely